platforms/documentation/docs/src/docs/userguide/running-builds/introduction/dependency_management_basics.adoc

```diff
@@ -45,11 +45,11 @@ androidGradlePlugin = "7.4.1"
 mockito = "2.16.0"
 
 [libraries]
-google-material = { group = "com.google.android.material", name = "material", version = "1.1.0-alpha05" }
-mockito-core = { module = "org.mockito:mockito-core", version.ref = "mockito" }
+googleMaterial = { group = "com.google.android.material", name = "material", version = "1.1.0-alpha05" }
```

Comment:
Hmmm what was the reasoning behind this change and not using the dashes? In the training we still use the dashes for mockito-core for example

Reply:
A user gave me feedback that in the introductory material it was too hard to understand that `test-this` could be accessed as `test.this`. So I made the intro a little easier to understand and kept this type of example in the more advanced sections of the manual.
Short answer: This was requested by our community.